examples/em-server-multi-tier-cronjob/analysis/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # python 3
-FROM emission/e-mission-server.dev.server-only:4.0.0
+FROM agutt/e-mission-server:ceo_ebike_project_2022-09-19--45-23
```

Comment:
@aGuttman you still need to fix this as well. Can you make sure that all my comments (even the ones that show a resolved, are actually resolved? Some might be just postponed to cleanup.

Otherwise looks good.

Reply:
Should I use `shankari/e-mission-server:master_2022-09-22--20-37` that was just created? A little unclear, because that also seems like a temp solution. I thought I would change it when we had new `emission/e-mission-server` images to use.

Reply:
@aGuttman yes, please use that for now so that we can merge this and the dashboard Dockerfile.
Then, as part of cleanup, we need to figure out how to import images properly
There are changes in addition to the org that we publish under.

For example, the jenkins deploy now allows us to specify the branch to use. How will that be reflected here?
Also, do we have to edit the file manually every time we push a new release to the server?

Reply:
Starting image changed to `shankari/e-mission-server:master_2022-09-22--20-37`. Builds, runs, browser gets redirected to https://www.nrel.gov/transportation/openpath.html as expected.